bpf/Makefile

```diff
@@ -131,24 +131,24 @@ bpf_overlay.o: bpf_overlay.ll
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_HOST_FIREWALL=1 \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_EXTERNAL_IP:-DENABLE_DSR \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_IPSEC:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_EXTERNAL_IP:-DENABLE_DSR \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_HOSTPORT \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_MASQUERADE \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_IPSEC:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_MASQUERADE \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_DSR \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_HOST_FIREWALL=1 \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_PREFILTER \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_PREFILTER:-DENABLE_HOST_FIREWALL=1 \
+	-DENABLE_IPV4=1:-DENABLE_IPV6=1:-DENCAP_IFINDEX=1:-DPOLICY_VERDICT_NOTIFY:-DENABLE_HOST_SERVICES_UDP:-DENABLE_HOST_SERVICES_TCP:-DENABLE_NODEPORT=1:-DENABLE_NODEPORT_ACCELERATION:-DENABLE_EXTERNAL_IP:-DENABLE_DSR:-DENABLE_DSR_HYBRID:-DENABLE_PREFILTER:-DENABLE_SESSION_AFFINITY:-DENABLE_HOST_FIREWALL=1
+
+MAX_HOST_OPTIONS = $(MAX_BASE_OPTIONS) -DENCAP_IFINDEX=1 -DENABLE_HOST_FIREWALL=1
 ifneq ("$(KERNEL)","49")
 MAX_HOST_OPTIONS += -DENABLE_HOST_SERVICES_UDP -DENABLE_HOST_SERVICES_TCP \
-	-DENABLE_NODEPORT -DENABLE_NODEPORT_ACCELERATION -DENABLE_EXTERNAL_IP -DENABLE_DSR \
+	-DENABLE_NODEPORT=1 -DENABLE_NODEPORT_ACCELERATION -DENABLE_EXTERNAL_IP -DENABLE_DSR \
```

Comment:
Did you consider using `=1` for _all_ options? I would personally prefer having a single way of doing this, especially since the macro names seem similar (e.g., `ENABLE_NODEPORT` vs `ENABLE_EXTERNAL_IP`)

Reply:
I did but was afraid it would bloat the lists of options even more. But maybe you're right and it's a safer option. Some of the options will expect non-1 values though.

Reply:
I've updated the branch to add `=1` to all options. Instead of doing it manually, I've extended the `subst` commands.